<?xml version="1.0"?>
<launch>

    <include file="$(find probot_gazebo)/launch/probot_anno_with_camera/probot_anno_with_camera_bringup_moveit.launch" />

    <node pkg="probot_demo" name="hand_camera" type="hand_camera.py" output="screen" />

    <!-- <node pkg="probot_demo" name="move" type="move.py" output="screen" /> -->
    
    <!-- <node pkg="probot_grasping" name="grasping_demo" type="grasping_demo.py" output="screen">
        <param name="table_length"          value="0.3"/>
        <param name="table_breadth"         value="0.3"/>
        <param name="pregrasp_x"            value="0.20"/>
        <param name="pregrasp_y"            value="-0.17"/>
        <param name="pregrasp_z"            value="0.46"/>
    </node> -->

</launch>
